src/components/DeploymentDetails.vue

```diff
@@ -17,7 +17,23 @@
         <DeploymentSchedulesFieldset :deployment="deployment" :schedules="deployment.schedules" @create="createDeploymentSchedule" @update="emit('update')" />
       </template>
     </p-key-value>
-    <slot />
+
+    <slot>
+      <p-key-value label="Triggers" :value="relatedAutomations">
```

Comment:
Using fallback content here for seamless compat w/ cloud which uses this slot (can remove that usage after this)